### PR TITLE
config: singular ANSIBLE_COLLECTIONS_PATH

### PR DESCRIPTION
**Pull request by jamescassell**
##### SUMMARY

Every other `*_PATH` setting in ansible is singular, and the traditional
`$PATH` variable is also singular despite containing a list of
directories.  Let's be consistent both internally and with POSIX
tradition.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME

config

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/config/base.yml [hunk 1]
```diff
@@ -234,8 +234,10 @@ COLLECTIONS_PATHS:
   type: pathspec
   env:
   - {name: ANSIBLE_COLLECTIONS_PATHS}
+  - {name: ANSIBLE_COLLECTIONS_PATH}
```
Comment (felixfontein) [inline lib/ansible/config/base.yml]: I'm not sure whether this works:
```suggestion
  - {name: ANSIBLE_COLLECTIONS_PATH, version_added: '2.10'}
```

Comment (bcoca) [reply lib/ansible/config/base.yml]: also, deprecate the old entry

Comment (felixfontein) [reply lib/ansible/config/base.yml]: In that case, the docs that reference the old one should be updated too. (I know of at least one mention in the dev guide. And maybe ansible-test is also using it somewhere.)

Comment (jamescassell) [reply lib/ansible/config/base.yml]: I've added commits to change all existing references to the singular version.  I've also deprecated the plural version.

**Review by nitzmahone — APPROVED** [review]

**Comment by felixfontein** [comment]
@jamescassell @nitzmahone @abadger thanks for creating, reviewing and merging this!

**Pull request merged**